src/components/MxGraph.tsx

```diff
@@ -37,9 +36,8 @@ class MxGraph extends React.Component<any, any> {
 
         this.nameToStandardCellDict = new Collections.Dictionary<string, any>();
         this.blockToCellDict = new Collections.Dictionary<Block, any>((b) => b.name);
-        this.subjectToBlockDict = new Collections.Dictionary<any, Block>();
-        this.triples = new Collections.Set<any>();
-        this.todoTriples = new Collections.Set<any>();
+        this.subjectToBlockDict = new Collections.Dictionary<string, Block>();
+        this.triples = new Collections.Set<Triple>((t) =>  t.subject + " " + t.predicate + " " + t.object);
```

Comment:
`t => t.subject + " " + t.predicate + " " + t.object` seems to recur. Consider factoring it out into its own function.

Reply:
Or just override `toString(): string` in `Triple`. That should work fine, too.